backend/util.py

```diff
@@ -8,6 +9,25 @@ def fetch_page(page_name: str) -> str:
         return page.read()
 
 
+def get_failed_message_by_HTTP_status(code) -> str:
+    """This method can help you to get HTTP status message by HTTP status code.
+    
+    It can pass the failed message we wrote, otherwise, return the default description of HTTP status code.
+    
+    Attribute:
+        code: HTTP status code.
+    """
```

Comment:
Docstring 的第一行總結函式的作用，介紹放在內文，可參考 [Google Python Style Guide: Comments and Docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)。

```python
    """Returns the common reason that causes the request to fail if defined, otherwise the default description of HTTP status code.
    
    This method helps you get HTTP status message with HTTP status code.
    """
```

`Args: code: HTTP status code.` 的區塊可以不用，因為從函式的 signature 中就能知道了。

Reply:
`get_failed_message_by_HTTP_status` 這個 function 已被廢棄，故這個修改建議應被標示為 `Resolve`.